Adding missing release notes (#2789)
kiegroup/kie-docs: doc-content/jbpm-docs/src/main/asciidoc/ReleaseNotes/Release.7.37.0.Final/search-by-variables.adoc

[id='search-by-variables-737']

= Search for Process Instances, Cases and Tasks

> The kie-server has been extended with more out-of-the-box operations to search for specific process instances, cases or tasks based on multiple constraints.  This introduces more flexible search criteria, where you can combine any number of these constraints, including for example process variables or task inputs and outputs.  You can also combine constraints from different elements, for example search for tasks where the associated process instances have specific variable values.  The implementation generates a SQL query on the fly based on the provided constraints and executed it on the underlying database.  The following operations were added to the kie-server apis: queryProcessesByVariables, queryCasesByVariables and queryUserTaskByVariables.

Reviewer: ```suggestion
The {KIE_SERVER} has been extended with more out-of-the-box operations to search for specific process instances, cases, or tasks based on multiple constraints.  This introduces more flexible search criteria, where you can combine any number of these constraints, including for example process variables or task inputs and outputs. You can also combine constraints from different elements, for example, search for tasks where the associated process instances have specific variable values.  The implementation generates a SQL query on the fly based on the provided constraints and executes on the underlying database. The following operations are added to the {KIE_SERVER} APIs including: queryProcessesByVariables, queryCasesByVariables, and queryUserTaskByVariables.
```
